packages/lit-dev-content/site/docs/components/decorators.md

```diff
@@ -73,25 +73,34 @@ In the future when decorators become a native web platform feature, this may no
 
 To use decorators with [TypeScript](https://www.typescriptlang.org/docs/handbook/decorators.html), enable the `experimentalDecorators` compiler option.
 
+You should also ensure that the `useDefineForClassFields` setting is `false`. Note, this should only be required when the `target` is set to `esnext` or greater, but it's recommended to explicitly ensure this setting is `false`.
+
 ```json
 "experimentalDecorators": true,
+"useDefineForClassFields": false,
 ```
 
 Enabling `emitDecoratorMetadata` is not required and not recommended.
 
 ### Using decorators with Babel  { #decorators-babel }
 
-If you're compiling JavaScript with [Babel](https://babeljs.io/docs/en/), you can enable decorators by adding the following plugins:
+If you're compiling JavaScript with [Babel](https://babeljs.io/docs/en/), you can enable decorators by adding the following plugins and assumptions:
```

Comment:
Using "assumptions" here is super jargony. You'd have to be pretty well-versed in Babel to know that there are config options called  "Compiler assumptions," and the average user  is likely to infer the generic meaning for assumptions here.

Suggest either just go with "plugins" and assume (heh) that the configuration steps that follow are part of adding the plugins, or say, "plugins and settings"

Reply:
Changed to "plugins and settings"